config/routes.rb

```diff
@@ -45,6 +45,8 @@
 
   match :faceted_search, controller: 'notices/search', action: 'index'
 
+  get '/twitter/international', to: redirect('/notices/search?utf8=%E2%9C%93&topics=international%252C+court+orders%252C+law+enforcement+requests%252C+government+requests&recipient_name=twitter&sort_by=')
```

Comment:
Not sure how well this would translate since these params are pretty hairy, but [default parameters](http://guides.rubyonrails.org/routing.html#defining-defaults) might be an easy way to pretty things up.

So something like this:

`get "/twitter/international", to: "notices#search", defaults: {topics: "...", recipient_name: "twitter"}`